airbyte-integrations/connectors/source-mixpanel/unit_tests/unit_test.py

```diff
@@ -1,26 +0,0 @@
-#
-# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
-#
-
-from datetime import date, timedelta
-
-import pendulum
-from source_mixpanel.streams import Export
-
-
-def test_date_slices():
```

Comment:
Should we add at least one mock server test to cover part of the Export stream?

Reply:
added [here](https://github.com/airbytehq/airbyte/pull/55189/files#diff-c7f124a55d2e4085ac9f360e06bc831def623a0cd0cef1deb5397d19ab098646R821)